doc/reference/ipdevpoll.rst

```diff
@@ -169,7 +135,7 @@ are reserved for synchronous communication with the PostgreSQL database
 backend. Even on a multi-core server, this means all of ipdevpoll's work is
 limited to a single core. Once ipdevpoll's workload grows beyond what a single
 core can handle, ipdevpoll can optionally run in a *multiprocess mode*, using
-the :option:`--multiprocess` option.
```

Comment:
I may be misreading your commit log here, but the warnings aren't because the keyword `option` doesn't exist, it's just that the docs don't have a properly tagged description of the actual command line options that, so this directive cannot find a cross reference link to generate.

I.e. there's a reason this is a warning and not an error. https://www.sphinx-doc.org/en/master/usage/domains/standard.html#directive-option

I assume our docs have opted to dump the output of `--help` instead of documenting all the options a second time in rst.

Reply:
I don't think we need to do anything atm.  There's no point in tagging these options with the `option` directive for cross-referencing until we actually have something to cross-reference.  I'm simply offering an explanation for future reference :)